src/stream/src/task/stream_manager.rs

```diff
@@ -283,28 +268,33 @@ impl LocalStreamManager {
 
     /// Force stop all actors on this worker, and then drop their resources.
     pub async fn stop_all_actors(&self) -> StreamResult<()> {
-        self.core.lock().await.stop_all_actors().await;
+        let actor_handles = self.core.lock().drain_actor_handles();
```

Comment:
I see that we inlined this method from `LocalStreamManagerCore`.

What's the general principle of the interface for `LocalStreamManagerCore` and `LocalStreamManager`?

What methods should be in `LocalStreamManagerCore`, and what methods should be in `LocalStreamManager`?

Reply:
Previously `LocalStreamManager` is more like a wrapper over `LocalStreamManagerCore`, and most logics and fields are implemented in `LocalStreamManagerCore`. 

This PR aims to make `LocalStreamManagerCore` only holds and manages some mutable owned states, such as actor map and actor handle, and `LocalStreamManager` will handle most async logic with no side effect, such as create actor.